domains/catalinaarciniegassaenz.json

```
@@ -0,0 +1,10 @@
+{
+  "domain": "catalinaarciniegassaenz.is-a.dev",
+  "owner": {
+    "email": "catalinaarciniegassaenz@gmail.com"
```

You need your username here.
```suggestion
    "username": "cascodenotch",
    "email": "catalinaarciniegassaenz@gmail.com"
```